kernel/src/vtpm/ptp/crb.rs

```diff
@@ -268,6 +311,51 @@ impl Tpm {
         self.reset()
     }
 
+    pub fn extend_svsm_version(&mut self) -> Result<()> {
+        let svsm_version_digests = if let Some(digests) = vtpm_get_svsm_version() {
+            digests
+        } else {
+            return Err(Error::ExtendSvsm);
+        };
+        let mut tpm2_command = [0u8; MAX_TPM_PCR_EXTEND_CMD_SIZE];
+        let mut tpm_cmd_size =
+            if let Ok(cmd_size) = tpm2_pcr_extend(&svsm_version_digests, 0, &mut tpm2_command) {
+                cmd_size
+            } else {
+                return Err(Error::ExtendSvsm);
+            };
+
+        let backend = vtpm_get_locked();
+
+        // Send TPM_STARTUP CMD
+        let mut tpm_startup_cmd = TPM_STARTUP_CMD;
```

Comment:
It might be better to move the startup related code to a function named `tpm_startup()`

Reply:
moved to function tpm_startup() in tdx/measurement.rs